site/app/views/NavigationView.php

```diff
@@ -661,7 +661,14 @@ private function getGradeButton(Gradeable $gradeable, int $list_section) {
             }
 
             if (!$gradeable->hasDueDate()) {
-                $progress_bar = $gradeable->getGradingProgress($this->core->getUser());
+
+                $cookie_string = "include_bad_submissions__" . $gradeable->getId() ;
+                $bad_submissions = array_key_exists($cookie_string, $_COOKIE) ? ($_COOKIE[$cookie_string]==="include" ? true : false) : false;
+
+                $cookie_string = "include_null_section__" . $gradeable->getId() ;
+                $null_section = array_key_exists($cookie_string, $_COOKIE) ? ($_COOKIE[$cookie_string]==="include" ? true : false) : false;
```

Comment:
Here's another good chance to clean up the code with the null coalescing operator:
```suggestion
                $null_section = ($_COOKIE[$cookie_string] ?? '') === "include";
```